XML/TestCases/Other.xml

```diff
@@ -78,4 +78,24 @@
       <SetupType>long-term</SetupType>
     </SetupConfig>
   </test>
+  <test>
+    <testName>NTTTCP-MONITOR-THROUGHPUT</testName>
+    <testScript>NTTTCP-MONITOR-THROUGHPUT.ps1</testScript>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\ntttcp_monitor_throughput.sh</files>
+    <Platform>Azure</Platform>
+    <TestParameters>
+      <param>seconds_of_runtime=NTTTCP_MONITOR_SECS_RUNTIME</param>
+      <param>monitor_interval=NTTTCP_MONITOR_INTERVAL</param>
+      <param>tolerance=NTTTCP_MONITOR_TOLERANCE</param>
+      <param>ntttcp_patch_file=NTTTCP_MONITOR_PATCH</param>
+      <param>ntttcp_version=NTTTCP_VERSION</param>
+    </TestParameters>
+    <Category></Category>
```

Comment:
I remember setting Tags to empty will make it always selected if -TestTag is specified.
Can you double check for these empty category, area, tags?

Reply:
Right, if '-TestTag' has any value from LISA command, it won't be a problem.  and because the <Category> and <Area> is also empty, it won't be selected, when user doesn't use any criteria at all. Anyway, I add a tag for this test case.  But I'm thinking the logic of Select-TestCase() is something weird and issue prone.  because -TestTag or -TestPriority is by default empty from LISAv2-Framework.psm1.  But the Matching behavior is not consistent between TestTag and TestPriority, once it has been giving some value from LISA commandline.

Reply:
This leads Get-LISAv2Statistics.ps1 utility output in the insufficient results. Please add values.

Reply:
will check, thanks.

Reply:
already added Tags, and this test is in Others.xml, which is already been excluded from Get-Statistics utility.

Reply:
Acked.